Source/WebCore/rendering/PathOperation.cpp

```diff
@@ -179,6 +179,7 @@ const std::optional<Path> RayPathOperation::getPath(const FloatRect& referenceRe
         length = lengthForContainPath(referenceRect, length, anchor, rotation);
     auto radians = deg2rad(toPositiveAngle(m_angle) - 90.0);
     auto point = FloatPoint(std::cos(radians) * length, std::sin(radians) * length);
+    path.moveTo({ });
     path.addLineTo(point);
```

Comment:
I guess it's better to be explicit, but `Path::addLineTo` will handle that.

Reply:
`moveTo()` was removed.